src/AsyncWebSocket.h

```diff
@@ -54,6 +54,8 @@
 #define DEFAULT_MAX_WS_CLIENTS 4
 #endif
 
+using ws_buffer_t = std::shared_ptr<std::vector<uint8_t>>;
```

Comment:
I'm not a fan of using `using` keyword like that in a header in a global scope. This can lead to naming conflicts because it isn't scoped. But I will gladly merge the order changed. Thanks!